test/hotspot/jtreg/compiler/lib/ir_framework/IRNode.java

```diff
@@ -2126,6 +2126,26 @@ public class IRNode {
         machOnlyNameRegex(VAND_NOT_L_MASKED, "vand_notL_masked");
     }
 
+    public static final String VAND_NOTI_VX = PREFIX + "VAND_NOTI_VX" + POSTFIX;
```

Comment:
Since these instructions are RISCV-specific, seems better to add a `RISCV_` prefix to make that explicit.
Consider names like `RISCV_VAND_NOTI_VX`, `RISCV_VAND_NOTL_VX`, `RISCV_VAND_NOTI_VX_MASKED` and `RISCV_VAND_NOTL_VX_MASKED`.